docker/config.hornet.json

```diff
@@ -243,4 +243,4 @@
       "workerCount": 0
     }
   }
-}
+}
```

Comment:
Please add missing newline.

```suggestion
}

```

Reply:
Why? My IDE seems to disagree.

Reply:
your IDE is probably not configured to add a blank line? Or Gh is lying to us.